src/pages/eth2/get-involved/index.js

```diff
@@ -0,0 +1,507 @@
+import React, { useContext } from "react"
+import { ThemeContext } from "styled-components"
+import Img from "gatsby-image"
+import styled from "styled-components"
+import { graphql } from "gatsby"
+import Card from "../../../components/Card"
+import Leaderboard from "../../../components/Leaderboard"
+import CalloutBanner from "../../../components/CalloutBanner"
+import Emoji from "../../../components/Emoji"
+import ProductCard from "../../../components/ProductCard"
+import ButtonLink from "../../../components/ButtonLink"
+import PageMetadata from "../../../components/PageMetadata"
+import CardList from "../../../components/CardList"
+import {
+  CardContainer,
+  Content,
+  Page,
+  GrayContainer,
+  Divider,
+  SloganGradient,
+} from "../../../components/SharedStyledComponents"
+
+const HeroContainer = styled.div`
+  padding-left: 0rem;
+  padding-right: 2rem;
+  padding-top: 8rem;
+  padding-bottom: 8rem;
+  width: 50%;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    padding-top: 2rem;
+    padding-left: 0rem;
+    width: 100%;
+  }
+`
+
+const StyledCard = styled(Card)`
+  flex: 1 1 30%;
+  min-width: 240px;
+  margin: 1rem;
+  padding: 1.5rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex: 1 1 30%;
+  }
+`
+
+const LeaderboardContainer = styled.div`
+  padding-left: 0rem;
+  padding-top: 2rem;
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    width: 100%;
+    margin-top: 2rem;
+  }
+`
+
+const HeroCard = styled.div`
+  display: flex;
+  justify-content: space-between;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+  }
+`
+
+const Subtitle = styled.div`
+  font-size: 24px;
+  line-height: 140%;
+  color: ${(props) => props.theme.colors.text200};
+  max-width: 480px;
+  margin-top: 1rem;
+`
+
+const Row = styled.div`
+  display: flex;
+  align-items: center;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    flex-direction: column;
+  }
+`
+
+const ReverseRow = styled.div`
+  display: flex;
+  align-items: center;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    flex-direction: column-reverse;
+  }
+`
+
+const Status = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 2rem;
+`
+
+const StyledCardContainer = styled(CardContainer)`
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+`
+
+const H2 = styled.h2`
```

Comment:
@ryancreatescopy I see we re-use these H2 styles a bunch - can we move this into SharedStyledComponents? Should this replace our existing H2 already there?